src/Matrix.jl

```diff
@@ -3558,6 +3558,8 @@ If `side` is `:right` and `flag` is true, $B$ is the right inverse of $A$
 i.e. $AB$ is the $n\times n$ unit matrix. If `side` is `:left` and `flag` is
 true, $B$ is the left inverse of $A$ i.e. $BA$ is the $m\times m$ unit matrix.
```

Comment:
While at it should also change this:
```suggestion
true, $B$ is a left inverse of $A$ i.e. $BA$ is the $m\times m$ unit matrix.
```
and then similarly in line 3557 for the right inverse.